src/FixedDataTableContainer.js

```diff
@@ -72,22 +75,24 @@ class FixedDataTableContainer extends React.Component {
 
   render() {
     const fdt = (
-        <FixedDataTable
-            {...this.props}
-            {...this.state}
-            scrollActions={this.scrollActions}
-            columnActions={this.columnActions}
-        />
+          <FixedDataTable
+              {...this.props}
+              {...this.state}
+              scrollActions={this.scrollActions}
+          />
     );
     // For backward compatibility, by default we render FDT-2 scrollbars
     if (this.props.defaultScrollbars) {
+      const { maxScrollX, scrollX } = this.state;
       return (
-          <ScrollContainer {...this.props}>
-            {fdt}
-          </ScrollContainer>
-      );
+          <PluginContext.Provider value={memoizeContext({ maxScrollX, scrollX })}>
+            <ScrollContainer {...this.props}>
+              {fdt}
+            </ScrollContainer>
+          </PluginContext.Provider>
+            );
     }
-    return fdt;
+    return <PluginContext.Provider value={{maxScrollX: this.state.maxScrollX}}>{fdt}</PluginContext.Provider>;
```

Comment:
please have an explicit method to compute the provider's value

Reply:
Done